docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+* Access operations runbooks so that I can resolve upcoming alerts
+* Access a monitoring dashboard to get an overview of the cloud Kubernetes environment and health
+
+== Iteration: Metrics
+
+As a user of the service I can perform the following actions:
+
+Collection::
+The service will automatically collect and aggregate metrics from my cloud Kubernetes environment and applications, providing a unified view of all metrics.
+
+Visualization::
+Metrics can be visualized in real-time, with interactive charts and graphs, providing insights into the behavior and performance of my cloud Kubernetes environment and the applications.
+
+Alarming::
+I can set up alarms to alert me when specific conditions are met, such as when an application experiences an error.
+
+Custom dashboards::
+I can create custom dashboards to view metrics that are most important to me, and to quickly identify areas of concern.
+
+
+== Iteration: Billing / Service
+
+As a user of the service I can perform the following actions:
+
+SLA::
+I can request a change for the SLA of my cloud Kubernetes environment
+
+Costs::
+I can view and access my running costs of the cloud Kubernetes environment
+
+As an engineer of the service I can perform the following actions:
+
+SLA::
+When a request for changing the SLA of a Kubernetes environment is approved, I can change the SLA.
+
+SLA::
+I can view the SLA of a Kubernetes environment per customer and per cluster
+
+
+== Iteration: Logs
+
+As a user of the service I can perform the following actions:
+
+Cluster Logs::
+I can access the logs of the Kubernetes control plane via a user-friendly interface
+
+Application Logs::
+I can access the logs of my applications via a user-friendly interface
+
+Log retention::
+I can configure the retention of the logs in the cloud provider's logging stack
+
+As an engineer of the service I can perform the following actions:
+
+Cluster Logs::
+I can access the logs of the Kubernetes control plane via a user-friendly interface
+
+
+== Iteration: Ingress Controller
+
+As a user of the service I can perform the following actions:
+
+Route traffic::
+I can define rules to route incoming traffic to specific services based on the URL path, host, or other criteria.
+
+Load balancing::
+I can configure the Ingress Controller to load balance incoming traffic across multiple replicas of my services to improve reliability and performance.
+
+SSL/TLS termination::
+I can configure the Ingress Controller to terminate SSL/TLS connections and handle encryption/decryption of incoming traffic, making it easier to secure my services.
+
+Custom configuration::
+I can customize the behavior of the Ingress Controller by providing custom configuration files, such as custom error pages or redirect rules.
+
+Monitoring and logging::
+I can monitor and log incoming traffic through the Ingress Controller, helping me to track traffic patterns and identify issues.
+
+Monitoring::
+I can monitor the expiration of my certificates
+
```

Comment:
We suggest to delete it from this Iteration, as every component can be monitored and we can mention it at the Monitoring Iteration.

Reply:
I agree that one doesn't really fit here. It would probably be a better fit for "User Workload Monitoring" which I'm unsure if this is yet another iteration or an additional component / add-on with its own features and iterations.